docker-compose.yml

```diff
@@ -22,14 +22,16 @@ services:
     image: postgres
     container_name: metadatadb
     environment:
-      POSTGRES_PASSWORD: password
+      POSTGRES_PASSWORD: pacifica
       POSTGRES_DB: pacifica_metadata
-      POSTGRES_USER: metadata
+      POSTGRES_USER: pacifica
```

Comment:
Revert these changes to docker-compose they don't have anything to do with what you are trying to fix.